### PR TITLE
GG-33046: Fix project description

### PR DESCRIPTION
**Pull request by isapego**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -120,6 +122,7 @@ def run_setup(with_binary=True):
         extras_require={
             'docs': requirements['docs'],
         },
+        license='Free for non-commercial use',
```
Comment (ptupitsyn) [inline setup.py:125]: I don't think this is correct:
https://docs.python.org/3/distutils/setupscript.html#additional-meta-data

File: setup.py [hunk 2]
```diff
@@ -112,6 +112,8 @@ def run_setup(with_binary=True):
         author='GridGain Systems',
         author_email='info@gridgain.com',
         description='GridGain binary client Python API',
+        long_description=long_description,
```
Comment (ptupitsyn) [inline setup.py:115]: Should we put the actual description text here?

**Review by ptupitsyn — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (isapego) [reply setup.py:115]: actual description text is taken from README.md and stored to `long_description` variable

File: setup.py (hunk 1 above)
Comment (isapego) [reply setup.py]: Fixed.

**Review by ptupitsyn — APPROVED** [review]

**Pull request merged**